html/elements/datalist.json

```diff
@@ -27,7 +27,7 @@
               "version_added": "9.5"
             },
             "opera_android": {
-              "version_added": null
+              "version_added": false
```

Comment:
```suggestion
              "version_added": true,
              "partial_implementation": true,
              "notes": "The dropdown menu containing available options does not appear in Opera for Android."
```

Reply:
Great! Thanks for the review and the suggestion. It is indeed true that the dropdown is missing, but the input block is being rendered.